python/morpheus/morpheus/_lib/include/morpheus/controllers/monitor_controller.hpp

```diff
@@ -314,11 +330,11 @@ std::string MonitorController<MessageT>::format_duration(std::chrono::seconds du
 }
 
 template <typename MessageT>
-std::string MonitorController<MessageT>::format_throughput(std::chrono::seconds duration,
+std::string MonitorController<MessageT>::format_throughput(std::chrono::microseconds duration,
                                                            size_t count,
                                                            const std::string& unit)
 {
-    double throughput = static_cast<double>(count) / duration.count();
+    double throughput = static_cast<double>(count) / duration.count() * 1000 * 1000;
```

Comment:
Just a small nit to avoid magic numbers with microseconds.

This also potentially makes it more accurate since there is only one operation (division). The internal ratio scaling of `chrono::duration` + `chrono::duration_cast` is really good.

```suggestion
    double time_in_seconds = std::chrono::duration_cast<std::chrono::duration<double>>(duration).count();
    double throughput = static_cast<double>(count) / time_in_seconds;
```

Reply:
Love this. Thanks!